tests/table/table_io/test_parquet.py

```diff
@@ -397,3 +399,25 @@ def test_read_parquet_file_missing_column_partial_read_with_pyarrow_bulk(tmpdir)
     read_back = read_parquet_into_pyarrow_bulk([file_path.as_posix()], columns=["x", "MISSING"])
     assert len(read_back) == 1
     assert tab.drop("y") == read_back[0]  # only read "x"
+
+
+@pytest.mark.parametrize(
+    "parquet_path", [Path(__file__).parents[2] / "assets" / "parquet-data" / "invalid_utf8.parquet"]
+)
+def test_parquet_read_string_utf8_into_binary(parquet_path: Path):
+    import pyarrow as pa
+
+    assert parquet_path.exists()
+
+    try:
+        read_parquet_into_pyarrow(path=parquet_path.as_posix())
+    except DaftCoreException:
+        # should throw an exception when `string_encoding` is not specified and is instead defaulted to `"utf-8"`
+        pass
+
+    read_back = read_parquet_into_pyarrow(path=parquet_path.as_posix(), string_encoding=None)
+
+    schema = read_back.schema
+    assert len(schema) == 1
+    assert schema[0].name == "invalid_string"
+    assert schema[0].type == pa.binary()
```

Comment:
Let's check the data in `read_back.schema` as well. I think I made it b`\x80\x80\x80` in this test data

Reply:
Ah okay. Will test to make sure that's indeed what I get.

Reply:
Updated.